lander/config.py

```diff
@@ -196,6 +237,96 @@ def _get_series_name(self, series):
         }
         return series_names.get(series.lower(), '')
 
+    @staticmethod
+    def _get_docushare_url(handle, validate=True):
+        """Get a docushare URL given document's handle.
+
+        Parameters
+        ----------
+        handle : `str`
+            Handle name, such as ``'LDM-151'``.
+        validate : `bool`, optional
+            Set to `True` to request that the link resolves by performing
+            a HEAD request over the network. `False` disables this testing.
+            Default is `True`.
+
+        Returns
+        -------
+        docushare_url : `str`
+            Shortened DocuShare URL for the document corresponding to the
+            handle.
+
+        Raises
+        ------
+        lander.exceptions.DocuShareError
+            Raised for any error related to validating the DocuShare URL.
+        """
+        logger = structlog.get_logger(__name__)
+        logger.debug('Using Configuration._get_docushare_url')
+
+        # Make a short link to the DocuShare version page since
+        # a) It doesn't immediately trigger a PDF download,
+        # b) It gives the user extra information about the document before
+        #    downloading it.
+        url = 'https://ls.st/{handle}*'.format(handle=handle.lower())
```

Comment:
The URL shortener is meant to be case insensitive so I don't think you need the `.lower()`.